src/test/java/StringCalculatorTest.java

```diff
@@ -0,0 +1,104 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.*;
+
+
+class StringCalculatorTest {
+
+    private StringCalculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new StringCalculator();
+    }
+
+    @ParameterizedTest(name = "{index}: {0} 계산식의 답은 {1} 입니다.")
+    @CsvSource(value = {"1 + 2 * 4 / 6 : 2", "2 + 3 - 7 : -2"}, delimiter = ':')
+    @DisplayName("연산에 대한 정확한 계산값을 리턴 하는지 테스트")
+    void calculateTest(String input, int answer) {
+
+        String[] array = calculator.enterCommand(input);
+        int result = calculator.parseInt(array[0]);
+
+        for(int i = 0; i < array.length - 2; i=i+2) {
+            String operator = array[i+1];
+            int num = calculator.parseInt(array[i+2]);
+
+            result = calculator.calculation(result, Operation.findOperator(operator), num);
+        }
+
+        assertThat(result).isEqualTo(answer);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"", "  "})
+    @NullSource
+    @DisplayName("빈 공백 문자일 경우, IllegalArgumentException 테스트")
+    void isEmpty_EnterCommand(String input) {
+        assertThatThrownBy(() -> calculator.enterCommand(input))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @ParameterizedTest(name = "{index}: ''{0}'' sign is not arithmetic operations.")
+    @ValueSource(strings = {"!", "@", "#", "^", "?", ">", "<", "(", ")"})
+    @DisplayName("사칙연산 기호가 아닌 경우, IllegalArgumentException 테스트")
+    void isTrueOrFalse_ArithmeticOperationTest(String sign) {
+        assertThatThrownBy(() -> calculator.calculation(2, Operation.valueOf(sign), 4))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    @DisplayName("+ 연산 기능 테스트")
+    void addTest(){
+        String[] inputs = calculator.enterCommand("5 + 3");
+        int num1 = calculator.parseInt(inputs[0]);
+        int num2 = calculator.parseInt(inputs[2]);
+
+        float result = calculator.calculation(num1, Operation.PLUS, num2);
+        assertThat(result).isEqualTo(8);
+    }
```

Comment:
```suggestion
    void addTest(){
        assertThat(calculator.calculation("5 + 3")).isEqualTo(8);
    }
```
이런방식으로 테스트가 가능하도록 바꿔볼 수 있을까요?

사실 문자열을 자르고 자른 문자열을 인수로 전달하고 하는 행위들은
이 API를 사용하는 사람들은 중요하게 생각하지 않을거 같아요.
(혹시나 이 서비스를 API로 제공할 수도 있다고 생각해 보면요😄)
`5 + 3`의 결과가 `8`로 잘 나오는게 제일 중요하지 않을까 생각됩니다